mastering-plone/relations.rst

```diff
@@ -68,15 +68,15 @@ For even more flexibility you can create your own `dynamic vocabularies <https:/
 Accessing and displaying related items
 --------------------------------------
 
-One would think that it would be the easiest approach to simply use the render-method of the default-widget like we did in the chapter "Views II: A Default View for “Talk”". Sadly that is wrong. Adding the approriate code to te template:
+One would think that it would be the easiest approach to simply use the render-method of the default-widget like we did in the chapter "Views II: A Default View for “Talk”". Sadly that is wrong. Adding the appropriate code to the template:
 
-..  code-block::html
+..  code-block:: html
 
     <div tal:content="structure view/w/evil_mastermind/render" />
 
 would only render the UIDs of the related items:
 
-..  code-block::html
+..  code-block:: html
```

Comment:
Remove one more extra space.

```
.. code-block:: html
```

Reply:
Same here.

The space after the colon is the fix. Without the space it don't work as expected. See: #388